tested/languages/bash/templates/encode.mako

```diff
@@ -0,0 +1,16 @@
+<%! from json import dumps %>
+function json_escape {
+    printf '%s' "$1" | python -c 'import json,sys; print(json.dumps(sys.stdin.read()))'
```

Comment:
What is the performance overhead of calling python for each value that's encoded?

Reply:
The overhead for each value that must be encoded is on average `14 ms`.
The timing is the result of testing encoding `Hello_0`...`Hello_9999`.

If you have suggestions for alternative JSON-encoding command, it could be usefull.

Reply:
Idea: https://stackoverflow.com/questions/10053678/escaping-characters-in-bash-for-json

Reply:
So e.g. an exercise with 100 function calls that all have a return value would add 100*14ms = ~1.5s to the execution right? It might be worth timing some of the alternatives from the StackOverflow answers to see if there is a faster one.

Reply:
Testing `jq` leads to an average execution time of `27 ms` which is lower that Python approach.

Reply:
Testing the PHP soluation leads to an average execution time of `11 ms`.

Reply:
I also tried binary compilation of the python script but that is even slow as `jq`.

Because PHP needs an additional package in the docker file, I suggest using the existing Python approach.